Module 5/Problem 1/problem1.json

```diff
@@ -0,0 +1,189 @@
+Input:
+
+
+{
+  "orders": [
+    {
+      "orderId": 1,
+      "orderDate": "2022-02-20T12:34:56Z",
+      "customer": {
+        "customerId": 1,
+        "firstName": "John",
+        "lastName": "Doe",
+        "email": "johndoe@example.com",
+        "address": {
+          "street": "123 Main St",
+          "city": "Anytown",
+          "state": "CA",
+          "zip": "12345"
+        }
+      },
+      "items": [
+        {
+          "itemId": 1,
+          "name": "Product A",
+          "quantity": 2,
+          "price": 10
+        },
+        {
+          "itemId": 2,
+          "name": "Product B",
+          "quantity": 1,
+          "price": 20
+        }
+      ]
+    },
+    {
+      "orderId": 2,
+      "orderDate": "2022-02-21T09:12:34Z",
+      "customer": {
+        "customerId": 2,
+        "firstName": "Jane",
+        "lastName": "Smith",
+        "email": "janesmith@example.com",
+        "address": {
+          "street": "456 Oak St",
+          "city": "Somewhere",
+          "state": "NY",
+          "zip": "67890"
+        }
+      },
+      "items": [
+        {
+          "itemId": 3,
+          "name": "Product C",
+          "quantity": 3,
+          "price": 15
+        }
+      ]
+    }
+  ]
+}
+
+
+  
+Transformation:
+
+[
+  {
+    "operation": "modify-overwrite-beta",
+    "spec": {
+      "orders": {
+        "*": {
+          "customer": {
+            "address": "=concat(@(1,address.street),' ',@(1,address.city), ' ', @(1,address.state), ' ',@(1,address.zip))",
+            "fullname": "=concat(@(1,firstName),' ',@(1,lastName))"
+          },
+          "itemTemp": "@(1,items)"
```

Comment:
It can be easily done without creating a duplicate of 'items' list. Please optimize this.